src/components/Header/Header.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
-import SideBar from './SideBar';
+import ConnectedSideBar from './SideBar';
```

Comment:
Naming connected components differently is not a common practice, and we do not get a lot of value from it.